includes/wikia/Wikia.php

```diff
@@ -166,6 +167,25 @@ public static function invalidateFavicon() {
 	}
 
 	/**
+	 * Return either a path to locally-uploaded Wiki.png file or a shared file taken from the code repo
+	 * (if there's no Wiki.png file uploaded on a wiki)
+	 *
+	 * @see SUS-1165
+	 *
+	 * @return string
+	 */
+	public static function getWikiLogoFullUrl() {
```

Comment:
As stated in the ticket: no wiki has `$wgLogo` set to a value different than the default.